python-backend/app/api/mines/variances/resources/variance.py

```diff
@@ -85,16 +83,16 @@ def put(self, mine_guid, variance_guid):
 
         data = self.parser.parse_args()
 
-        core_user_guid = data.get('inspector_guid')
-        if core_user_guid:
-            core_user = CoreUser.find_by_core_user_guid(core_user_guid)
-            if not core_user:
+        inspector_party_guid = data.get('inspector_party_guid')
+        if inspector_party_guid:
+            inspector = Party.find_by_party_guid(inspector_party_guid)
+            if not inspector:
                 raise BadRequest('Unable to find new inspector.')
 
-            variance.inspector_id = core_user.core_user_id
+            variance.inspector_party_guid = inspector_party_guid
```

Comment:
Should we validate this person is infact an inspector?

Reply:
Added a check.